### ISSUE TITLE
Epic Fight Mod difficulties with multiplayer

### ISSUE THREAD

**Issue by Gigarus** [Yesssssman/epicfightmod#799]
I'm having issues joining my server hosted by apexhostingmc while epic fight mod for 1.16.5 is installed. As soon as I join, it instantly kicks me/crashes. I'm using a modpack, but these mods are all compatible with each other on singleplayer, including epic fight mod.
Here is an example of the crash report. https://mclo.gs/HhLQQ2A

EDIT: Sometimes it crashes before I join, due to a ticking entity caused by this mod? Here is the crash report for that. https://mclo.gs/vqbIQsw

**Comment by Yesssssman** [comment]
Please try in 16.5.12 version. There was a massive memory usage issue when installing many other mods and it was solved in the latest.

**Comment by Gigarus** [comment]
I updated the server's version to 16.5.12, and it crashed a few seconds after launching. Here was the crash report: https://mclo.gs/ANXEvnW

**Comment by Yesssssman** [comment]
It looks like your environment variable for Java is set to 16. To launch the 1.16.5 server, you have to set it to Java 8.

**Comment by Gigarus** [comment]
I set it to that version of java, and I received another crash with ticking entities. 

apparently this mod runs smoothly on 1.17.1, but whenever I try to host it with apexhosting, it crashes; here is the log report:
https://mclo.gs/cb00jyu

**Comment by Gigarus** [comment]
I got it running by using a previous version of forge,

but I ran into an other client-sided bug which makes everything my character hold tiny, and I've tested this with the mod alone without any other mods, and it's still occurring. Here's what it looks like: https://imgur.com/a/59Br7gp
I'm positive this is a client-sided issue because apexhosting didn't have this happen to them, I cleared my config and tried running this mod on 36.2.0 forge for 1.16.5, but it still gave me the glitch, and I also tried using previous versions of the mod but the glitch still occurred. Any reason why this glitch is happening?
